src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+  transition-duration: 0.3s;
+  transition-property: transform;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__title &:hover {
+    color: #34568b;
+  }
+  &__image {
+    margin-bottom: 40px;
+    object-fit: contain;
+  }
+
+  &__title {
+    color: $main-accent;
+    margin-bottom: 4px;
+    transition-duration: 0.3s;
+    transition-property: color;
+  }
+
+  &__code {
+    font-weight: $font-normal;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    align-self: flex-start;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__review {
+    font-weight: $font-normal;
+    font-size: 10px;
+    color: $main-accent;
+    text-align: right;
+    line-height: 14px;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
```

Comment:
leave here only margins beacuse it dedicates position of this element in card, but display flex and justify-content would be better to have at your separate block - 'price-box'

Reply:
Yes, that's make more sense ;-)